src/styles/blocks/container.scss

```diff
@@ -0,0 +1,8 @@
+
+.container {
+  display: flex;
+  justify-content: space-between;
```

Comment:
it would be better to change this property starting from left side. now It looks not appropriate by adding 2 cards 
![image](https://user-images.githubusercontent.com/22824431/222738954-05022a79-6045-4713-8c90-a4d8cbffedcf.png)
```suggestion
  justify-content: left;
```